tools/update-prod-static

```diff
@@ -46,6 +46,19 @@ subprocess.check_call(['./tools/minify-js'] +
                       (['--prev-deploy', prev_deploy] if prev_deploy else []),
                       stdout=fp, stderr=fp)
 
+# Copy the KaTeX files outside node_modules
+subprocess.check_call(['mkdir', '-p',
+                       'prod-static/serve/node_modules/katex/dist/'],
+                      stdout=fp, stderr=fp)
+
+subprocess.check_call(['cp', 'node_modules/katex/dist/katex.css',
+                       'prod-static/serve/node_modules/katex/dist/'],
+                      stdout=fp, stderr=fp)
+
+subprocess.check_call(['cp', '-R', 'node_modules/katex/dist/fonts',
+                       'prod-static/serve/node_modules/katex/dist/fonts'],
```

Comment:
These paths work when building a release tarball but not for `upgrade-zulip-from-git`.  Use
e.g. 'os.path.join(settings.STATIC_ROOT, 'node_modules/katex/dist/fonts')`